qulice-maven-plugin/src/test/java/com/qulice/maven/DefaultMavenEnvironmentTest.java

```diff
@@ -128,4 +128,30 @@ public final void passPathsWithWhitespaces()  throws Exception {
             Matchers.notNullValue()
         );
     }
+
+    /**
+     * DefaultMavenEnvironment can produce empty collection when no matches
+     * with checker.
+     * @throws Exception If something wrong happens inside
+     * @todo #676:30min Ideally integration test required to reproduce NPE.
```

Comment:
@vkuchyn No, it is not duplication. `#676:30min` will be swallowed by PDD parser and only the rest will be presented to the issue resolver in GitHub. Sometimes original issue it is not important, but here it is crucial.